src/utils/auth_utils.js

```diff
@@ -33,6 +34,32 @@ const firebaseConfig = {
 const app = initializeApp(firebaseConfig);
 const auth = getAuth(app);
 
+const NPOBackend = axios.create({
+  baseURL: 'http://localhost:3001', // Replace baseURL with NPO-specific URL
+  withCredentials: true,
+});
+
+const addRoleToCookies = async cookies => {
+  const user = await NPOBackend.get(`/users/${auth.currentUser.email}`);
```

Comment:
would be better if you use the user's firebase ID rather than their email because the firebase ID will always be unique